operations-stdlib/src/iosMain/kotlin/Format.kt

```diff
@@ -0,0 +1,58 @@
+import kotlinx.cinterop.cstr
+import operation.StandardLogicOperation
+import platform.Foundation.NSString
+import platform.Foundation.stringWithFormat
+import utils.asList
+import utils.secondOrNull
+
+actual object Format : StandardLogicOperation {
+    actual override fun evaluateLogic(expression: Any?, data: Any?): Any? {
+        return with(expression.asList) {
+            val format = firstOrNull().toString()
+            val args = secondOrNull().asList
+
+            runCatching { format.formatString(args) }
+                .fold(
+                    onSuccess = { it },
+                    onFailure = { null }
+                )
+        }
+    }
+
+    private fun String.formatString(args: List<Any?>): String {
+        val regEx = "%[\\d|.]*[sdf]|[%]".toRegex()
+        val formats = regEx.getFormats(this)
+        val rawStrings = this.split(regEx).toMutableList()
+
+        if (args.isEmpty() || formats.isEmpty()) {
+            return this
+        }
+
+        var formattedText = args.formatText(rawStrings, formats)
+
+        // Add the rest of text if left
+        formattedText += rawStrings.joinToString("")
```

Comment:
maybe move this line to formatText function? 
```
fold("") { acc: String, argument ->
            ...
        } + rawStrings.joinToString("")
```
So here you can remove `var formattedText` and just return 
```
    private fun String.formatString(args: List<Any?>): String {
        ....
        return args.formatText(rawStrings.toMutableList(), formats)
    }
```